src/include/network/lan/address.rb

```diff
@@ -1553,8 +1555,6 @@ def AddressDialog
         LanItems.vlan_id = Builtins.tostring(
           Ops.get_integer(@settings, "VLAN_ID", 0)
         )
-      elsif LanItems.type == "br"
-        LanItems.bridge_ports = Ops.get_string(@settings, "BRIDGE_PORTS", "")
```

Comment:
Removing this to `StoreBridge` avoid current ports in case that the `bridge ports tab` is not visited.
## Previously

![bridge_ports_removed](https://cloud.githubusercontent.com/assets/7056681/18738659/9555f36e-8095-11e6-814c-df654db6845b.gif)

## After the fix

![bridge_ports_removed](https://cloud.githubusercontent.com/assets/7056681/18738980/4164f20c-8098-11e6-9552-c11ed036e471.gif)